applications/launchpad/backend/src/api/wallet_api.rs

```diff
@@ -64,6 +65,13 @@ pub async fn wallet_balance() -> Result<WalletBalance, String> {
 
 #[tauri::command]
 pub async fn wallet_events(app: AppHandle<Wry>) -> Result<(), String> {
+    loop {
```

Comment:
we are subscribing to wallet_events on application start (and call this command)
that means this loop will run from launchpad start until the user successfully starts the wallet

is this really a good approach to have this loop running all the time?
maybe we could start the stream when backend starts wallet container?
that way frontend would just listen to events and backend would open the stream when it's ready, not when frontend requests it